multidimensional-pod-autoscaler/KEP.md

```diff
@@ -0,0 +1,519 @@
+# KEP-5342: Multi-dimensional Pod Autoscaler
```

Comment:
This is a made up KEP ID.

Reply:
For this question, we are unsure if Kubernetes is the right target. We would like to start with the autoscaler as the code base mostly reuses existing VPA and HPA.